airbyte-webapp/src/components/ui/NumberBadge/NumberBadge.module.scss

```diff
@@ -0,0 +1,43 @@
+@use "../../../scss/colors";
+@use "../../../scss/fonts";
+
+.circle {
+  height: 20px;
+  width: fit-content;
+  min-width: 20px;
+  border-radius: 10px;
+  padding: 0 4px;
+  text-align: center;
+  overflow: hidden;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  line-height: normal;
+
+  &.default {
+    background: colors.$dark-blue-100;
+  }
+
+  &.green {
+    background: colors.$green;
+    color: colors.$white;
+  }
+
+  &.red {
+    background: colors.$red;
+    color: colors.$white;
+  }
+
+  &.blue {
+    background: colors.$blue;
+    color: colors.$white;
+  }
+}
+
+.number {
+  font-family: fonts.$primary;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 10px;
```

Comment:
All those styles could be moved to the `circle` class above instead, so we wouldn't need this class at all.